packages/scss/src/components/_menu.scss

```
@@ -123,18 +134,23 @@
 .menu-list-item-action {
 	&.is-active,  // legacy syntax
 	&[aria-current="page"] {
-		color: _color('primary', '700');
+		color: _component('menu.default-palette.700');
+
+		.label {
+			background-color: _component('menu.default-palette.100');
+			color: _component('menu.default-palette.700');
+		}
 
 		&::after {
-			background-color: _color('primary', '700');
+			background-color: _component('menu.default-palette.700');
 			transform: scale(1);
 		}
 	}
 
 	&.is-disabled {
```

add [disabled] when there are buttons
```suggestion
	&.is-disabled, &[disabled] {
```